src/contexts/ConfigurationContext.tsx

```diff
@@ -59,9 +59,11 @@ interface Config {
 
   // Summary
   config_summary_default_language?: string;
+  config_summary_default_style?: string;
   config_summary_num_results?: number;
   config_summary_num_sentences?: number;
   config_summary_prompt_name?: string;
+  config_summary_styled_prompt?: string;
```

Comment:
This should be a boolean, since it's a flag:

```
config_summary_styled_prompt?: boolean;
```

Reply:
So this is actually a string in all our configs (e.g in "mmr" and "enable_source_filters"). Why would it be different here? In all those cases, we'd be using isTrue(X) as above.